src/main/java/frc/robot/subsystems/elevator/ElevatorConstants.java

```diff
@@ -36,18 +47,48 @@ public class ElevatorConstants { // TODO: check real values
     public static final LoggedTunableNumber KP = new LoggedTunableNumber("kp");
     public static final LoggedTunableNumber KI = new LoggedTunableNumber("ki");
     public static final LoggedTunableNumber KD = new LoggedTunableNumber("kd");
+    public static final LoggedTunableNumber KG = new LoggedTunableNumber("kg");
 
     public static void initConstants() {
         switch (Constants.CURRENT_MODE) {
             case REAL:
                 KP.initDefault(0.8);
                 KI.initDefault(0.0);
                 KD.initDefault(0.0);
+                KG.initDefault(0.0);
             case SIM:
             case REPLAY:
                 KP.initDefault(43.0);
                 KI.initDefault(0.0003);
                 KD.initDefault(0.05);
         }
+
+        MAIN_MOTOR_CONFIGURATION.withFeedback(new FeedbackConfigs().withSensorToMechanismRatio(GEAR_RATIO))
+                .withSlot0(
+                        new Slot0Configs()
+                                .withKP(ElevatorConstants.KP.get())
+                                .withKI(ElevatorConstants.KI.get())
+                                .withKD(ElevatorConstants.KD.get())
+                                .withKG(ElevatorConstants.KG.get()))
+                .withMotorOutput(new MotorOutputConfigs().withInverted(MAIN_INVERT))
+                .CurrentLimits
+                .withStatorCurrentLimitEnable(true)
+                .withSupplyCurrentLimitEnable(true)
+                .withStatorCurrentLimit(CURRENT_LIMIT)
+                .withSupplyCurrentLimit(CURRENT_LIMIT);
+
+        AUX_MOTOR_CONFIGURATION.withFeedback(new FeedbackConfigs().withSensorToMechanismRatio(GEAR_RATIO))
+                .withSlot0(
+                        new Slot0Configs()
+                                .withKP(ElevatorConstants.KP.get())
+                                .withKI(ElevatorConstants.KI.get())
+                                .withKD(ElevatorConstants.KD.get())
+                                .withKG(ElevatorConstants.KG.get()))
+                .withMotorOutput(new MotorOutputConfigs().withInverted(AUX_INVERT))
+                .CurrentLimits
+                .withStatorCurrentLimitEnable(true)
+                .withSupplyCurrentLimitEnable(true)
+                .withStatorCurrentLimit(CURRENT_LIMIT)
+                .withSupplyCurrentLimit(CURRENT_LIMIT);
```

Comment:
1. Both configurations are 98% identical. Why not use the same object?
2. Stator and supply current mean different things -- do they need to have the same value (I don't know)?
3. Missing Motion Magic Expo configurations,
4. I don't think you need to configure almost anything for the auxiliary motor, except maybe inversion.

Reply:
1. for the inverted value
2. They do need to have the same value
3. 9134a00ec9884204f91bc83d393ccd26c07c3320
4. I wasn't sure but I think you are right 618a56c9100f26ca30d8e1eb3193a6f865e3cae8

Reply:
2. If they were needed to be the same value there wouldn't have been two different configurations. I suggest searching Chief Delphi or the FRC Discord channel.

Reply:
current stator- is the output - the current flow through the motor
current supply- is the input - the current flow that come from the PDP
It makes sense that their limit is the same

Reply:
They would be the same if you would lose no energy, and only when running at 100%. This post explains why they are different: https://www.chiefdelphi.com/t/current-limiting-talonfx-values/374780/10?u=dan

Reply:
Yes this is what I read, so it still makes sense that the limit is the same...

Reply:
Changed to Fault

Reply:
The calculation shown in the example shows that in lower torques movements the output current can be larger than the input current. This is very relevant to the elevator, because when we climb we need a lot of torque but a low voltage, so the stator current may be much higher than the supply current. If the supply current limit would be low enough to prevent a breaker from tripping, and the stator current limit would be the same, the motors might not produce enough torque to lift the whole robot.
I hope that's clear, I learned about the differences only yesterday. And again, if both limits would always be the same, there would have been only one current limit.

Reply:
48f2ea0742a9cabcfa8f41fd109d9648d20318a3